### PR TITLE
added a formula to build docker 1.6.2

### PR DESCRIPTION
**Pull request by shtouff**

### CODE REVIEW

File: docker162.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class Docker162 < Formula
+  homepage "https://www.docker.com/"
+  url "https://github.com/docker/docker.git",
+      :tag => "v1.6.2"
+
+  bottle do
```
Comment (DomT4) [inline docker162.rb]: Remove this block. We'll add it on `pull`.

File: docker162.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+class Docker162 < Formula
+  homepage "https://www.docker.com/"
+  url "https://github.com/docker/docker.git",
+      :tag => "v1.6.2"
```
Comment (DomT4) [inline docker162.rb]: Needs a `:revision` line as well.

**Comment by shtouff** [comment]
Hi, 

I've removed the 'bottle' block and added a revision, as instructed.

Have a good day,

**Comment by DomT4** [comment]
Looks like there's just the one change left:

```
==> brew style docker162

1 file inspected, no offenses detected
Error: 1 problem in 1 formula

==> audit problems
homebrew/versions/docker162:
 * Formula should have a desc (Description).
```

**Comment by DomT4** [comment]
Ping?

**Comment by shtouff** [comment]
Hi,

I'm in holidays, no access to a computer until next wednesday.

I'll keep you posted.

Regards,

Le lun. 13 juil. 2015 06:27, Dominyk Tiller <notifications@github.com> a
écrit :

**Comment by DomT4** [comment]
Thanks. No worries, just wanted to check the PR wasn't dead.

**Comment by shtouff** [comment]
Hello,
I added a desc field, as requested:

    $ brew style docker162
    [...]
    1 file inspected, not offenses detected

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-versions/commit/0a81cfa419000f9818765a6394b94766db2530f2. Thank you for your contribution to Versions, we appreciate it! :smiley_cat:.

Couple of future nits, new formulae should have their commit message styled `docker162 1.6.2 (new formula)` or similar, and when there's an existing formula in Homebrew/homebrew that installs the same binary/binaries as the new package we need to add a `conflicts_with` line so people can't link them on top of each other. Thanks!

**Pull request closed without merging**